src/main/fc/fc_msp.c

```diff
@@ -1663,14 +1667,23 @@ static mspResult_e mspFcProcessInCommand(uint8_t cmdMSP, sbuf_t *src)
     case MSP_SET_VTX_CONFIG:
         {
             const uint16_t tmp = sbufReadU16(src);
-            const uint8_t band    = (tmp / 8) + 1;
-            const uint8_t channel = (tmp % 8) + 1;
 
             if (vtxCommonGetDeviceType() != VTXDEV_UNKNOWN) {
+
+                if (tmp < 1000) {  //value is band and channel
+                const uint8_t band    = (tmp / 8) + 1;
+                const uint8_t channel = (tmp % 8) + 1;
                 uint8_t current_band=0, current_channel=0;
                 vtxCommonGetBandAndChannel(&current_band,&current_channel);
                 if ((current_band != band) || (current_channel != channel))
                     vtxCommonSetBandAndChannel(band,channel);
+                }
+                else {  //value is frequency in MHz
+                    uint16_t currentFreq;
+                    vtxCommonGetFrequency(&currentFreq);
+                    if (currentFreq != tmp)
```

Comment:
can you fix the `if` statements so that the conditional code is wrapped in `{}`'s?

e.g. always use this style.
```
if (condition) {
  doSomething();
}
```
```